### PR TITLE
feat!: Deprecate AWS SQL policies

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: plugins/source/aws/policies/README.md [hunk 1]
```diff
@@ -1,78 +1,9 @@
...
+## Deprecated:
 
-- [AWS CIS V1.2.0](./cis_v1.2.0/policy.sql)
-- [AWS PCI DSS v3.2.1](./pci_dss_v3.2.1/policy.sql)
-- [AWS Foundational Security Best Practices](./foundational_security/policy.sql)
-- [AWS Public Egress](./public_egress/policy.sql)
-- [AWS Publicly Available](./publicly_available/policy.sql)
-- [AWS Unused Resources](./unused_resources/policy.sql)
+CloudQuery SQL Policies for AWS has been deprecated all queries have been moved to the `/policies_v1` directory. Going forward all CloudQuery Policies will continue to be available in the [CloudQuery Hub](https://hub.cloudquery.io/addons/transformation). The Policies listed via the Hub have many benefits including:
+- Built on top of DBT for ease of extensibility and integration with other tools
```
Comment (yevgenypats) [inline plugins/source/aws/policies/README.md:7]: Maybe comment to old commit - old policies still available here in the git history.

Comment (hermanschaaf) [inline plugins/source/aws/policies/README.md:7]:
```suggestion
- Built on top of dbt for ease of extensibility and integration with other tools
```

File: plugins/source/aws/policies/README.md [hunk 2]
```diff
@@ -1,78 +1,9 @@
...
+- Multi database support (PostgreSQL, BigQuery, Snowflake)
```
Comment (hermanschaaf) [inline plugins/source/aws/policies/README.md:8]:
```suggestion
- Multi-database support (PostgreSQL, BigQuery, Snowflake)
```

Comment (bbernays) [reply plugins/source/aws/policies/README.md]: completed in https://github.com/cloudquery/cloudquery/pull/15487/commits/26eb52212f571662ec1fe7c352a7fc16448fdb23
and 
https://github.com/cloudquery/cloudquery/pull/15487/commits/b3a2a6e137cb470e07c2648c060bdd8dbbfb4d1f

File: plugins/source/aws/policies/README.md (hunk 1 above)
Comment (bbernays) [reply plugins/source/aws/policies/README.md]: completed in https://github.com/cloudquery/cloudquery/pull/15487/commits/26eb52212f571662ec1fe7c352a7fc16448fdb23 and https://github.com/cloudquery/cloudquery/pull/15487/commits/b3a2a6e137cb470e07c2648c060bdd8dbbfb4d1f

File: plugins/source/aws/policies/README.md [hunk 3]
```diff
@@ -1,78 +1,9 @@
...
-## Policies and Compliance Frameworks Available
+## Deprecated:
 
-- [AWS CIS V1.2.0](./cis_v1.2.0/policy.sql)
-- [AWS PCI DSS v3.2.1](./pci_dss_v3.2.1/policy.sql)
-- [AWS Foundational Security Best Practices](./foundational_security/policy.sql)
-- [AWS Public Egress](./public_egress/policy.sql)
-- [AWS Publicly Available](./publicly_available/policy.sql)
-- [AWS Unused Resources](./unused_resources/policy.sql)
+CloudQuery SQL Policies for AWS has been deprecated all queries have been moved to the `/policies_v1` directory. Going forward all CloudQuery Policies will continue to be available in the [CloudQuery Hub](https://hub.cloudquery.io/addons/transformation). The Policies listed via the Hub have many benefits including:
```
[the remark this answers is not in the document]
Comment (bbernays) [reply plugins/source/aws/policies/README.md]: completed in https://github.com/cloudquery/cloudquery/pull/15487/commits/26eb52212f571662ec1fe7c352a7fc16448fdb23 and https://github.com/cloudquery/cloudquery/pull/15487/commits/b3a2a6e137cb470e07c2648c060bdd8dbbfb4d1f

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**